charts/fybrik/values.yaml

```diff
@@ -50,6 +50,26 @@ global:
   # zerolog verbosity level 
   # ref: https://github.com/rs/zerolog#leveled-logging
   loggingVerbosity: -1
+  # True if the target deployment is OpenShift. Otherwise false. Default value is false.
+  openshift: false
+  # Pod Security Context. This is the default setting for all pods, and can be
+  # overwritten by a specific podSecurityContext settings.
+  # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
+  podSecurityContext:
+    runAsNonRoot: true
+    runAsUser: 10001
```

Comment:
On openshift you should support running as an arbitrary user so runAsUser should not be specified.

Reply:
we don't set `runAsUser` for OpenShift

Reply:
But you need to set all the other security context on openshift. Just not the runAsUser.  If running on openshift the security context will not be set. Or am I missing something?
```
      {{- if not .Values.global.openshift }}
      securityContext:
        {{- mergeOverwrite (deepCopy .Values.global.podSecurityContext) .Values.manager.podSecurityContext | toYaml | nindent 8 }}
      {{- end }}
```